test/iterative_verifier_test.go

```diff
@@ -82,6 +105,30 @@ func (t *IterativeVerifierTestSuite) TestVerifyOncePass() {
 	t.Require().Equal("", result.Message)
 }
 
+func (t *IterativeVerifierTestSuite) TestVerifyCompressedOncePass() {
+	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.SourceDB)
+	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.TargetDB)
+
+	result, err := t.verifier.VerifyOnce()
+	t.Require().NotNil(result)
+	t.Require().Nil(err)
+	t.Require().True(result.DataCorrect)
+	t.Require().Equal("", result.Message)
+}
+
+func (t *IterativeVerifierTestSuite) TestVerifyCompressedMismatchOncePass() {
```

Comment:
nitpick but `Mismatch` seems a bit ambiguous, how about something like `SameDecompressedDataButDifferentHash` to be more explicit?